### DISCUSSION TITLE
Failed to send response to webhook localhost url

### DISCUSSION THREAD

**Discussion by NavinAT** [gotenberg/gotenberg#829] [Q&A]
Hai,

I am evaluating the Gotenberg for our needs with my local environment. I'm trying to use the webhook feature, my WEB API is deployed in the local **IIS server** (webhook URL - https://localhost:443/AppName/api/pdfresult). The API is not getting called and got some error in the docker desktop terminal.
 
![image](https://github.com/gotenberg/gotenberg/assets/45414603/1c91102f-51cd-4909-bfbf-5cc24d854090)

I could able to communicate with my API in Postman, Couldn't find the reason for this issue. Help me to resolve this...

**Comment by gulien** [comment]
Hello @NavinAT,

localhost is the… localhost of your Gotenberg container.

If you want to reach a local app, your best option is to put it in the same docker-compose stack as your Gotenberg instance.

**Comment by NavinAT** [comment]
Hello @gulien 

Thanks for the clarifications. i achieved it by communicating my application with the IP address 😁.

**Comment by Morzaram** [comment]
I've written an explanation on how to fix this here: https://github.com/gotenberg/gotenberg/discussions/886#discussioncomment-11748292
